content/news/022/index.md

```diff
@@ -123,6 +123,47 @@ multiple turn lanes, U-turns, and stop signs much better.
 [A/B Street]: https://github.com/a-b-street/abstreet
 [@dabreegster]: https://twitter.com/CarlinoDustin
 
+### [Outer Wonders]
+
+![Animation showing Bibi, the main protagonist of Outer Wonders,
+starting from the top entrance of a puzzle, rolling from obstacle
+to obstacle, leaning on on bushes, flowers and trees to reach
+the bottom exit of the puzzle](outer-wonders.gif)
+
+[Outer Wonders] is a colorful, pixel art, puzzle-based adventure game
+developed by [Utopixel] where you play as Bibi, a cute round monkey who
+enjoys rolling in straight lines. Explore a whimsical nature where
+altering the environment is key to progress, and solve puzzles to protect
+its wonders.
+
+Outer Wonders can be downloaded for Linux and Windows from [itch.io].
+
+May was mostly dedicated to code cleaning, small improvements, as well as
+blogging and community building. Updates of the month include:
+
+- Refactored UI code to streamline menu stacking and ease the implementation
+  of an upcoming options menu.
+- Added support for menu navigation using the D-Pad alongside the existing
+  analog stick support.
+- Published a blog post about building Outer Wonders for Linux/itch.io
+  ([english], [french]).
+- Posted weekly puzzles [#16], [#17], [#18] and [#19] on social media
+  for players wishing to give puzzles a try prior to downloading the game.
+
+_Discussions:
+[/r/rust_gamedev](https://www.reddit.com/nfgxjl),
+[Hacker News](https://news.ycombinator.com/item?id=27244017)_
+
+[Outer Wonders]: https://utopixel.itch.io/outer-wonders
+[Utopixel]: https://utopixel.games
+[itch.io]: https://utopixel.itch.io/outer-wonders
```

Comment:
Very minor suggestion - it might be a good idea to change this link to have a more unique ID, to ensure other sections can't accidentally override your link with a different URL 😅

You can keep the link text as is by doing `[itch.io][link-id-goes-here]`.